service/hook/endpoint.go

```diff
@@ -171,10 +172,22 @@ func HTTPHandler(w http.ResponseWriter, r *http.Request) {
 	respondWith := hookCommon.TransformResponseInputModel{
 		Errors:                  []string{},
 		SuccessTriggerResponses: []bitriseapi.TriggerAPIResponseModel{},
+		SkipResponses:           []bitriseapi.SkipAPIResponseModel{},
```

Comment:
to conform to the naming of the other related params rename this to `SkippedTriggerResponses`